ramls/request.json

```diff
@@ -378,6 +378,11 @@
           "$ref": "override-blocks.json"
         }
       }
+    },
+    "searchIndex": {
+      "description": "Request fields used for search",
+      "type": "object",
+      "$ref": "requestSearchIndex.schema"
```

Comment:
should be requestSearchIndex.json?

Reply:
Also consistent filenames. The others are hyphen-separated.

Reply:
Done